src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -697,7 +644,26 @@ bool emberAfWindowCoveringClusterUpOrOpenCallback(app::CommandHandler * commandO
     {
         Attributes::TargetPositionTiltPercent100ths::Set(endpoint, WC_PERCENT100THS_MIN_OPEN);
     }
+
+    Delegate * delegate = GetDelegate(endpoint);
+    if (delegate)
+    {
+        if (HasFeature(endpoint, Feature::kPositionAwareLift))
+        {
+            LogErrorOnFailure(delegate->HandleMovement(WindowCoveringType::Lift));
+        }
+        else if (HasFeature(endpoint, Feature::kPositionAwareTilt))
```

Comment:
why is there an else here?

What if both features exist? Should we not be passing a set of what movement to handle or is the spec saying that only one of lift or tilt features have to exist?

Reply:
Spec seems to say "*At least one of the Lift and Tilt features SHALL be supported." but I cannot find any text that says they are mutually exclusive.

But logically it does seem like they may be exclusive. Maybe just a comment suffices saying that the SDK expects only one of the features being set.

I think we can add the comment as a followup (or change to bit flags)

Reply:
no there are not exclusive , must have at least one or both but cannot have none

Reply:
This is incorrect we must have both running at the same time